cpp/src/parquet/column_writer_test.cc

```diff
@@ -418,23 +419,29 @@ TYPED_TEST(TestPrimitiveWriter, RequiredRLE) {
 TYPED_TEST(TestPrimitiveWriter, RequiredBitPacked) {
   this->TestRequiredWithEncoding(Encoding::BIT_PACKED);
 }
+*/
+
+TEST_F(TestValuesWriterInt32Type, RequiredDeltaBinaryPacked) {
+  this->TestRequiredWithEncoding(Encoding::DELTA_BINARY_PACKED);
+}
 
-TYPED_TEST(TestPrimitiveWriter, RequiredDeltaBinaryPacked) {
+TEST_F(TestValuesWriterInt64Type, RequiredDeltaBinaryPacked) {
   this->TestRequiredWithEncoding(Encoding::DELTA_BINARY_PACKED);
 }
```

Comment:
> Should we add a roundtrip test in the arrow_reader_writer_test.cc file to make sure this is covered?

I don't see where (and if) arrow_reader_writer_test.cc tests encodings. I've enabled this test as this is probably what you're suggesting.

Reply:
Ah, I misunderstood which line you were commenting due to GitHub interface